dragonphy/fpga_models/analog_slice.py

```diff
@@ -29,8 +30,8 @@ def __init__(self, filename=None, **system_values):
                              real_type=get_dragonphy_real_type())
 
         # Random number generator seeds (defaults generated with random.org)
-        m.add_digital_param('jitter_seed', width=32, default=46428)
-        m.add_digital_param('noise_seed', width=32, default=8518)
+        m.add_digital_input('jitter_seed', width=32)
```

Comment:
random seeds are now generally wired as signals, rather than as parameters.  i'm not sure if this is necessary, but i have seen bugs in Vivado's handling of parameter arrays in the past